src/io/CMakeLists.txt

```diff
@@ -32,3 +32,6 @@ target_link_libraries(rs_TFBroadcaster rs_core)
 
 rs_add_library(rs_ResultAdvertiser src/ResultAdvertiser.cpp)
 target_link_libraries(rs_ResultAdvertiser rs_core)
+
+rs_add_library(rs_loadPointCloud src/LoadPointCloud.cpp)
+target_link_libraries(rs_loadPointCloud rs_core)
```

Comment:
I would consider renaming the annotator. Eventually we would like to be able to load not only pcd files but also depth and rgb images. Something like FileReader perhaps or you could also suggest something.